.cspell.json

```diff
@@ -503,6 +503,7 @@
     "springboot",
     "errmsg",
     "shelljs",
+    "compatability",
```

Comment:
It is better to fix it where it is happening:
https://github.com/novuhq/novu/blob/e54550062cc5fe2b48f5ba353ff7f2298a69174a/apps/api/e2e/compile-email-template.e2e.ts#L143
https://github.com/novuhq/novu/blob/e54550062cc5fe2b48f5ba353ff7f2298a69174a/apps/widget/public/iframeResizer.contentWindow.js#L872

Reply:
Yeah this fix came from it complaining about a comment that now is removed thu.